icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -1066,7 +1066,13 @@ class PySession:
     @property
     def store(self) -> PyStore: ...
     def merge(self, other: PySession) -> None: ...
-    def commit(self, message: str, metadata: dict[str, Any] | None = None) -> str: ...
+    def commit(
+        self,
+        message: str,
+        metadata: dict[str, Any] | None = None,
+        rebase_with: ConflictSolver | None = None,
+        rebase_tries: int = 5,
```

Comment:
```suggestion
        rebase_tries: int = 1000,
```

In my experience, 5 is almost never going to be enough. If you invoke this option, I think you want to to basically keep trying until it truly hits a conflict.